src/modules/emote_menu/stores/emote-storage.js

```diff
@@ -64,9 +64,15 @@ class EmoteStorage extends SafeEventEmitter {
   }
 
   trackHistory(emote) {
-    const id = getEmoteIdFromProvider(emote.id, emote.provider.id);
+    let emoteCanonicalId = null;
 
-    let emoteHistory = this.emoteStore.usageHistory[id];
+    try {
+      emoteCanonicalId = getEmoteIdFromProvider(emote.id, emote.provider.id);
+    } catch (e) {
+      emoteCanonicalId = emote.id;
```

Comment:
curious: why are the emotes in `favorites` and `frecents` classified as those providers? shouldn't `emote.provider` be their original provider? see the code at `src/modules/emote_menu/stores/index.js` line 129, 133, where emote is an existing reference.